app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -64,6 +60,20 @@ class ProfileEditActivityPresenter @Inject constructor(
       showDeletionDialog(profileId)
     }
 
+    editViewModel.profile.observe(
```

Comment:
This looks more clean!! What's you though on this? Same for the below one. 
```kotlin
editViewModel.profile.observe(
      activity, Observer {
        activity.title = it.name
      }
    )
```

Reply:
DONE